JavaScript/Task_Day2/Partition.js

```diff
@@ -0,0 +1,20 @@
+//https://www.codewars.com/kata/525a037c82bf42b9f800029b/train/javascript
+
+//Partition on
+
+// partition the items array so that all values for which pred returns true are
+// at the end, returning the index of the first true value
+
+function partitionOn(pred, items) {
+  var temp_arr_1 = items.filter((val, index, items) => {
```

Comment:
you can remove index and items params from the filter, since they are not used anywhere inside that filter method